catalogs/catalog-jdbc-postgresql/src/test/java/com/datastrato/gravitino/catalog/postgresql/integration/test/service/PostgreSqlService.java

```diff
@@ -88,11 +89,8 @@ public void close() {
     }
   }
 
-  public void executeQuery(String sql) {
-    try (Statement statement = connection.createStatement()) {
```

Comment:
You should not remove the try-with-resource statement

Reply:
But this is in the test module , and at [comment](https://github.com/datastrato/gravitino/pull/3195#discussion_r1593211518) above, you say:
> Oh, since this belongs to the test module, we can just throw an exception in the method signature.

I think we can throw an exception in the method signature here.

Reply:
the try-with-resource is to be able to close the `statement`, not to catch the exception

Reply:
```java
try (Statement statement = connection.createStatement()) {
// ....
}
```

Reply:
Oh, I got it ! 
I've reverted it.
Reference:
- [Java – Try with Resources](https://www.baeldung.com/java-try-with-resources)